flink-formats/flink-hadoop-bulk/src/test/java/org/apache/flink/formats/hadoop/bulk/AbstractFileCommitterTest.java

```diff
@@ -245,7 +240,7 @@ private void verifyCommittedFiles(
         for (String targetFileName : targetFileNames) {
             Path targetFilePath = new Path(basePath, targetFileName);
             assertThat(fileSystem.exists(targetFilePath))
-                    .as("Committed file should exists: " + targetFileName)
+                    .as("Committed file should exist: " + targetFileName)
```

Comment:
```suggestion
                    .as(() -> "Committed file should exist: " + targetFileName)
```
we can use suppliers in order to concat string only in case of failure and do not dot it for success cases